src/base_authentication_app_skeleton/src/actions/mixins/auth/allow_guests.cr

```diff
@@ -1,4 +1,4 @@
-module Auth::SkipRequireSignIn
+module Auth::AllowGuests
```

Comment:
Rather than saying what it is skipping, I'm thinking it is better to say what it should be used for, allowing guests.